soc/README.md

```diff
@@ -0,0 +1,33 @@
+Summer of Code
+=====================
+
+Organization Admins
+-------------------
+
+If you need help with anything Summer of code related, you can file an issue or contact one of the admins below:
+
+- Michelle Noorali ([@michellenoorali](https://github.com/michelleN)\): https://twitter.com/michellenoorali
+- Thomas Rampelberg ([@grampelberg](https://github.com/grampelberg)\): https://twitter.com/grampelberg
+
+Communication
+-------------
+
+Please reach out to us on the on the [SMI slack](https://aka.ms/smi/slack).
+Please don't use DMs unless strictly necessary as doing so both has the potential of overwhelming project maintainers and others with similar questions lose the benefit of public discussion.
+
+Project Ideas
+-------------
+
+#### Istio Adapter
+
+- Description: Converting from the SMI APIs to Istio specific configuration is a critical part of supporting every service mesh that has been released. There has been some work done on this already, but more needs to be done. The work involves setting up CI, writing end to end tests, updating the adapter and APIs to work with the latest version of Istio and documenting the contribution process for the project.
+- Recommended Skills: Golang, Kubernetes
+- Mentor(s): Michelle Noorali (@michellenoorali)
+- Issues: https://github.com/deislabs/smi-adapter-istio/issues/22, https://github.com/deislabs/smi-adapter-istio/issues/70, https://github.com/deislabs/smi-adapter-istio/issues/81
```

Comment:
CI is failing because these are bare URLs. Didn't even know lint would complain about things like that. We'll want to update these as text with links attached.